tasks/installation.yml

```diff
@@ -1,5 +1,11 @@
 ---
 
+- name: ensure apt-transport-https is installed
+  apt:
+    name: apt-transport-https
+    state: latest
```

Comment:
I'd suggest to use `present` to prevent uncontrolled updates of `apt-transport-https` in order to be idempotent. :)